src/components/Authentication/Login.vue

```diff
@@ -0,0 +1,110 @@
+<template lang="pug">
+.login-widget
+  v-alert(
+      dismissible,
+      transition="scale-transition",
+      v-for="err in alerts.errors",
+      :key="err",
+      :value="!!err",
+      type="error") {{ err }}
+  v-form(@submit.prevent="login", ref="login")
+    v-container
+      v-text-field(
+          v-model="username",
+          label="Username or e-mail",
+          autofocus,
+          :rules="nonEmptyRules",
+          prepend-icon="$vuetify.icons.user",
+          type="text")
+      v-text-field(
+          v-model="password",
+          type="password",
+          label="Password",
+          :rules="nonEmptyRules",
+          prepend-icon="$vuetify.icons.lock")
+      .submit-area
```

Comment:
We can get rid of the style block if we change this to a `v-layout(row)` and then add `v-spacer` between the two buttons.